ts/components/conversation/UpdateGroupDialog.tsx

```diff
@@ -88,16 +89,25 @@ export class UpdateGroupDialog extends React.Component<Props, State> {
   public render() {
     const checkMarkedCount = this.getMemberCount(this.state.friendList);
 
-    const titleText = `${this.props.titleText} (Members: ${checkMarkedCount})`;
+    let titleText = `${this.props.titleText} (Members: ${checkMarkedCount})`;
 
     const okText = this.props.okText;
     const cancelText = this.props.cancelText;
 
-    const noFriendsClasses =
+    let noFriendsClasses =
       this.state.friendList.length === 0
         ? 'no-friends'
         : classNames('no-friends', 'hidden');
 
+    // alternatively, we can go back to const and use more trinary operators
```

Comment:
Not super important, but in that case I would even leave the variables undefined, and then add an `else` to that `if` (there is no reason why private groups should be the default case).